packages/containers/package.json

```diff
@@ -146,6 +148,7 @@
   "dependencies": {
     "memoize-one": "^4.0.0",
     "react-immutable-proptypes": "^2.1.0",
-    "react-router": "^3.2.0"
+    "react-router": "^3.2.0",
```

Comment:
This is a dep of @talend/react-cmf-router, you can drop it

Reply:
no it s used by the SidePanel explicitly